timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/NamespacedClients.java

```diff
@@ -71,11 +84,29 @@ static NamespacedClients from(String namespace, ProxyFactory proxyFactory) {
 
     @Value.Derived
     default TimelockService timelockService() {
+        LockTokenUnlocker unlocker = new LegacyLockTokenUnlocker(namespacedConjureTimelockService());
+
+        LockLeaseService lockLeaseService = LockLeaseService.create(
```

Comment:
Looks like we just moved this out from RemoteTimelockServiceAdapter, so it's not created there but passed as a dep.